scripts/core/superdesk-api.d.ts

```diff
@@ -889,6 +889,7 @@ declare module 'superdesk-api' {
     export interface ISuperdeskGlobalConfig {
         // FROM SERVER
         default_language: string;
+        disallowed_characters: Array<string>;
```

Comment:
Can you add a comment that it only applies to `slugline`? It's a little bit configurable, but still hardcoded.